src/main/java/com/gtnewhorizon/gtnhlib/ClientProxy.java

```diff
@@ -112,4 +115,17 @@ public void addMessageToChat(IChatComponent componentText) {
     public void printMessageAboveHotbar(String message, int displayDuration, boolean drawShadow, boolean shouldFade) {
         AboveHotbarHUD.renderTextAboveHotbar(message, displayDuration, drawShadow, shouldFade);
     }
+
+    @Override
+    public void sendMessageAboveHotbar(
+            EntityPlayerMP player,
+            IChatComponent chatComponent,
+            int displayDuration,
+            boolean drawShadow,
+            boolean shouldFade) {
+        if (FMLCommonHandler.instance().getEffectiveSide() == Side.CLIENT) {
```

Comment:
I can't think of a situation where you have a EntityPlayerMP but is on logical client.

Reply:
Indeed, `EntityPlayerMP` requires `MinecraftServer` and `WorldServer` for instantiation